fairground/fairground.toml

```diff
@@ -7,6 +7,11 @@ Host = "127.0.0.1"
 [API]
   [API.GRPC]
     Hosts = [
+      "n01.testnet.vega.xyz:3007",
+      "n02.testnet.vega.xyz:3007",
+      "n03.testnet.vega.xyz:3007",
+      "n04.testnet.vega.xyz:3007",
+      "n05.testnet.vega.xyz:3007",
```

Comment:
3007 is a data-node port, and there is no data-node up and running on nodes n01-n05.
We can either:
- remove nodes n01-n05
- or use port `3002` - the one from vega core